packages/tldraw/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable deprecation/deprecation */
```

Comment:
let's not do this for the entire file (b/c it might hide something useful down the line).

let's prefer doing a lint disable/enable around the lines that are relevant (then it's clearer later what it meant for).

Reply:
slightly wanna push back on this one. in general, I would agree that we don't want to disable lint rules for entire files. however, I think that this specific file and this specific rule are worth the exception. This file just lists everything in the library. It doesn't do anything, or make use of any of those exports. It should never contain anything other than the big long list of exports.

The `deprecation/deprecation` rule is to stop us from using deprecated things. But this file doesn't use anything. It just exports it - and it _has_ to export it. This is the one file where referencing deprecated things is actually legit and expected - if they're deprecated but not exported, then we might as well just delete them.

Let me know what you think - if you'd still rather do something more targeted lmk and i'll be happy to.